app/models/concerns/hyrax/admin_set_behavior.rb

```diff
@@ -32,6 +32,18 @@ def self.default_set?(id)
         id == DEFAULT_ID
       end
 
+      # Creates the default AdminSet and an associated PermissionTemplate with workflow
+      def self.create_default!
```

Comment:
Please use `# rubocop:disable Lint/HandleExceptions` as we had before (and remove the exception from the rubocop config`  this ensure that only this method is allowed this exception and not the whole of the file.